yoti-sdk-api/src/main/java/com/yoti/api/client/identity/ShareSessionQrRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.yoti.api.client.identity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ShareSessionQrRequest {
+
+    @JsonProperty(Property.TRANSPORT)
+    private final String transport;
+
+    @JsonProperty(Property.DISPLAY_MODE)
+    private final String displayMode;
```

Comment:
I think @laurent-yoti has decided to leave this out of the SDKs for now. The service will use default values anyway (neither of these are mandatory) so probably best to keep SDKs aligned and drop thus for now (unless there's a compeling reason why they should be included)

cc\ @AdrienVioletYoti

Reply:
it was mentioned in the sync-up as well. I'll drop it.

Reply:
We have not really mentioned per say in the sync up, but as we showed the implementation we did, this is indeed ignored right now, and we let Connect use its default value.